object_image[] object_list
sensor_msgs/Image image
